packages/app/components/activity/activity-dropdown.tsx

```diff
@@ -17,6 +20,9 @@ import { useMyInfo } from "app/hooks/api-hooks";
 import { useReport } from "app/hooks/use-report";
 import { useUser } from "app/hooks/use-user";
 
+const isWeb = Platform.OS === "web";
+const DropdownMenuSeparator = isWeb ? Fragment : ShowtimeDropdownMenuSeparator;
```

Comment:
Why not simply remove the `<DropdownMenuSeparator />`? IIRC we don't need them on native

Reply:
Oh don't we? I didn't know that, sure that would be definitely better. I will handle it now 👍🏽